apidocs/overview.md

```diff
@@ -0,0 +1,438 @@
+
+<blockquote>
+    <h3><span>❗️</span>Rate limiting and SDKs</h3>
+    <p>Our SDKs are never rate limited. Our SDKs do not use the API endpoints defined here-- we use a different set of approaches (streaming / server-sent events as well as a global CDN) to ensure availability to the routes used by our SDKs.</p>
+    <p>The client-side ID is safe to embed in untrusted contexts-- it's designed for use in client-side JavaScript.</p>
+</blockquote>
+
+## Global rate limits
+
+Authenticated requests are subject to a global limit-- this is the maximum number of calls that can be made to the API per ten seconds. All personal access tokens on the account share this limit, so exceeding the limit with one access token will impact other tokens. Calls that are subject to global rate limits will return the headers below:
+
+| Header name | Description |
+| ----------- | ----------- |
+| `X-Ratelimit-Global-Remaining` | The maximum number of requests the account is permitted to make per ten seconds. |
+| `X-Ratelimit-Reset` | The time at which the current rate limit window resets in epoch milliseconds. |
+
+We do not publicly document the specific number of calls that can be made globally. This limit may change, and we encourage clients to program against the specification (relying on the two headers defined above) rather than hardcoding to the current limit.
+
+## Route-level rate limits
+
+Some authenticated routes have custom rate limits. These also reset every ten seconds. Any access tokens hitting the same route share this limit, so exceeding the limit with one access token may impact other tokens. Calls that are subject to route-level rate limits will return the headers below:
+
+| Header name | Description |
+| ----------- | ----------- |
+| `X-Ratelimit-Route-Remaining` | The maximum number of requests to the current route the account is permitted to make per ten seconds. |
+| `X-Ratelimit-Reset` | The time at which the current rate limit window resets in epoch milliseconds. |
+
+A *route* represents a specific URL pattern and verb. For example, the [Delete environment](doc:delete-environment) endpoint is considered a single route, and each call to delete an environment counts against your route-level rate limit for that route. 
+
+We do not publicly document the specific number of calls that can be made to each endpoint per ten seconds. These limits may change, and we encourage clients to program against the specification (relying on the two headers defined above) rather than hardcoding to the current limits.
```

Comment:
```suggestion
We do not publicly document the specific number of calls that can be made to each endpoint per ten seconds. These limits may change, and we encourage clients to program against the specification, relying on the two headers defined above, rather than hardcoding to the current limits.
```